truss/config/trt_llm.py

```diff
@@ -28,10 +28,10 @@ class TrussTRTLLMQuantizationType(str, Enum):
     SMOOTH_QUANT = "smooth_quant"
     FP8 = "fp8"
     FP8_KV = "fp8_kv"
+    FP8_ROWWISE = "fp8_rowwise"
```

Comment:
was the builder updated to support this? has this flow been tested?

Reply:
I'm going to remove this – the builder was updated to support this but it was for 405B, which has been scoped out